FEATURE_GATES.md

```diff
@@ -69,7 +69,7 @@ Features that reach GA and over time become stable will be removed from this tab
 | FillIDs                    | `true`  | Beta  | 3.0.0  | TBD   |
 | RewriteURIs                | `false` | Alpha | 2.12.0 | TBD   |
 | KongServiceFacade          | `false` | Alpha | 3.1.0  | TBD   |
-| SanitizeKonnectConfigDumps | `false` | Alpha | 3.1.0  | TBD   |
+| SanitizeKonnectConfigDumps | `true`  | Alpha | 3.1.0  | TBD   |
```

Comment:
The only comment I'd have is to reconsider the `alpha` stage for this flag. Having this enabled by default, I'd be surprised to see it having `alpha` label on it.

Reply:
Well, yeah, good point :)